src/Altinn.Platform/Altinn.Platform.Storage/IntegrationTest/Clients/InstanceClient.cs

```diff
@@ -83,9 +87,10 @@ public async Task<List<Instance>> GetInstancesForOrg(string org, int size = 100)
         /// <returns>the instance just created</returns>
         public async Task<Instance> PostInstances(string appId, int instanceOwnerPartyId)
         {
-            string requestUri = $"{versionPrefix}/instances?appId={appId}";
+            string requestUri = $"{_versionPrefix}/instances?appId={appId}";
 
-            HttpResponseMessage response = await client.PostAsync(hostName + requestUri, new Instance()
+            _client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", _validToken);
+            HttpResponseMessage response = await _client.PostAsync(_hostName + requestUri, new Instance()
```

Comment:
Consider using PostAsJsonAsync instead of calling .AsJson() at the end.

Reply:
These clients will be removed once all integration tests in storage have been upgraded. I moved them from storage.interface to be able to add token to the existing tests.. Not sure how much time we should spend fixing this code.. ?